policy/release/tasks_test.rego

```diff
@@ -639,7 +639,27 @@ test_expired_slsa_v1 if {
 
 	expected := {{
 		"code": "tasks.unsupported",
-		"msg": `Task "task" is used by pipeline task "task" is or will be unsupported as of 2000-01-01T00:00:00Z.`,
+		"msg": `Task "task" is used by pipeline task "task" is or will be unsupported as of 2000-01-01T00:00:00Z. Upgrade to a newer version of the Task.`,
+		"term": "task",
+	}}
+
+	lib.assert_equal_results(tasks.deny, expected) with input.attestations as attestation
+		with data["pipeline-required-tasks"] as {"generic": []}
+		with data["task-bundles"] as _trusted_tasks
+}
+
+test_expired_with_custom_message if {
+	attestation := _slsav1_attestations_with_tasks({}, [object.union(
+		_task("task"),
+		{"invocation": {"environment": {"annotations": {
+			tasks._expires_on_annotation: "2000-01-01T00:00:00Z",
+			tasks._expiry_msg_annotation: "The Task has been discontinued.",
```

Comment:
I noticed the other test case is kinda quadrupled - `[slsav0_2, slsav1] X [already expired, expires in the future]`

It didn't seem necessary to cover the expiry message for all 4 cases, but LMK if that's incorrect